service/controller/controller.go

```diff
@@ -13,19 +13,19 @@ import (
 	"github.com/giantswarm/prometheus-meta-operator/pkg/project"
 )
 
-type TODOConfig struct {
+type ControllerConfig struct {
```

Comment:
```suggestion
type ClusterConfig struct {
```

Since this watches CAPI CR's I'd name the controller cluster which aligns with SOTA.

https://github.com/giantswarm/aws-operator/blob/ac69ef376548cfeff50577003331e6264a9e7c0c/service/controller/cluster.go#L28

Reply:
I'd disagree with this. Sure, we watch CAPI CRs, but we're not controlling clusters - at best, it'd be prometheus controller or some such.

Reply:
True, that makes sense. 👍

Reply:
@rossf7 is right. It's about what it watches and not what it controls. In the newer lingo that would probably be "handler" which would probably make more sense.